test/entrypoint.sh

```diff
@@ -22,5 +88,13 @@ check_return_code
 # Deprecated Image
 curl -X 'GET' 'https://catalog.redhat.com/api/containers/v1/repositories/registry/registry.access.redhat.com/repository/rhscl%2Fnodejs-8-rhel7' -H 'accept: application/json' > image.json
 
+# Test parse_hacbs_test_output
+test_parse_hacbs_test_output(){
+  conftest test --namespace optional_checks --policy $POLICY_PATH/image/inherited-labels.rego label.json --output=json > unittest.json
+  HACBS_TEST_OUTPUT=
+  parse_hacbs_test_output sanity_label_check conftest unittest.json
+  [ "$(echo $HACBS_TEST_OUTPUT | jq -r '.result')" == "SUCCESS" ] && exit 0 || exit 1
```

Comment:
Looks like calling `exit 0` exits the entrypoint script completely, maybe we can have something like `echo "test_parse_hacbs_test_output PASSED"` instead?

Reply:
Yes, `exit 0` will quit. Thanks!